doc/user_guide/postgresql_user_guide.md

```diff
@@ -47,7 +47,7 @@ The SQL statement below creates the adapter script, defines the Java class that
 --/
 CREATE OR REPLACE JAVA ADAPTER SCRIPT ADAPTER.JDBC_ADAPTER AS
   %scriptclass com.exasol.adapter.RequestDispatcher;
-  %jar /buckets/<BFS service>/<bucket>/virtual-schema-dist-12.0.0-postgresql-3.0.0.jar;
+  %jar /buckets/<BFS service>/<bucket>/virtual-schema-dist-12.0.0-postgresql-3.0.1.jar;
```

Comment:
This will be misleading for customers, because version 3.0.1 is not released... The only way to fix this is to create a release...